src/effects/VST3/VST3Effect.cpp

```diff
@@ -1200,11 +1149,7 @@ bool VST3Effect::LoadPreset(const wxString& path)
       return false;
    }
 
-   if(!Vst::PresetFile::loadPreset(
-      fileStream,
-      FUID::fromTUID(mEffectClassInfo.ID().data()),
-      mEffectComponent.get(),
-      mEditController.get()))
+   if ( ! VST3Wrapper::LoadPreset(fileStream, FUID::fromTUID(mEffectClassInfo.ID().data())) )
```

Comment:
I still have a feeling that putting `mEffectClassInfo` into the wrapper base class might make nicer simplifications of usage... but I don't insist

Reply:
I forgot about that, now done.